logs_ingest/main.py

```diff
@@ -191,7 +192,19 @@ def parse_to_json(text):
     try:
         event_json = json.loads(text)
     except Exception:
-        event_json = json.loads(text.replace("\'", "\""))
+        try:
```

Comment:
I guess this one will work:
`try:
    event_json = json.loads(text.replace('\\\'', '').replace("\'", "\"").replace("\n", ""), strict=False)
except Exception:
    logging.exception(`